capabilitytypes/radon.capabilities.datapipeline/ConnectToPipeline/ConnectToPipeline.tosca

```diff
@@ -0,0 +1,9 @@
+tosca_definitions_version: tosca_simple_yaml_1_3
+
+capability_types:
+  ConnectToPipeline:
```

Comment:
The type names should always contain the respective namespace. See the other types in the particles repo.

Reply:
Solved this issue in 3 files. Did not find other occurrences.